aws-params-reader-policy/main.tf

```diff
@@ -30,6 +29,10 @@ data "aws_iam_policy_document" "policy" {
     actions   = ["kms:Decrypt"]
     resources = [data.aws_kms_alias.parameter_store_key.target_key_arn]
   }
+  statement {
+    actions = ["ssm:DescribeParameters"]
+    resources = ["arn:aws:ssm:${var.region}:${data.aws_caller_identity.current.account_id}:parameter/*"]
+  }
```

Comment:
DescribeParameters doesn't require a Resource value. This was a relief because we can still limit the reach of other perms, like `ssm:GetParameters`, `ssm:GetParametersByPath`, `ssm:GetParameter`, and `ssm:GetParameterHistory`.